content/en/apps/guides/performance/purging.md

```diff
@@ -106,6 +106,8 @@ This function takes four parameters:
 - `contact`, the contact document of a patient or other contact who has reports about them.
 - `reports`, an array of all reports for that subject that are present on the server.
 - `messages`, an array of sms messages that the contact has sent or received
+- `chtScriptApi` the CHT API that provides CHT-Core Framwework's functions to toher parts of the app. More info on thea API can be found [here](https://docs.communityhealthtoolkit.org/apps/reference/_partial_cht_api/)
```

Comment:
```suggestion
- `chtScriptApi` the CHT API that provides CHT-Core Framwework's functions to other parts of the app. More info on the API can be found [here](https://docs.communityhealthtoolkit.org/apps/reference/_partial_cht_api/)
```

Reply:
Also for the links withing the cht-docs we use this notation: `[tasks]({{<ref "apps/reference/tasks" >}})` it will let the app resolve the link automatically in case we change the domain.